.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -34,6 +34,13 @@ jobs:
     env:
       ROX_PRODUCT_VERSION: ${{ matrix.version }}
     steps:
+    - name: Recover docker image cache space
```

Comment:
This job doesn't have a checkout step, so running all of the `preamble` didn't make sense here